src/rgw/rgw_process.cc

```diff
@@ -13,8 +13,21 @@
 #include "rgw_loadgen.h"
 #include "rgw_client_io.h"
 
+#include <stdio.h>
+#include <stdlib.h>
+#include <string>
+#include <iostream>
+#include <curl/curl.h>
+
 #define dout_subsys ceph_subsys_rgw
 
+
+size_t WriteCallback(char *contents, size_t size, size_t nmemb, void *userp)
```

Comment:
though there are tons of examples in src/rgw that contradict this, in c++, "pointer to char" is char* and "pointer to void" is void*